core/example/solvers/CMakeLists.txt

```
@@ -0,0 +1,30 @@
+if(Cabana_ENABLE_Cuda)
+  add_definitions(-DCUDA_ENABLE)
+  find_package(CUDA)
```

```suggestion
  find_package(CUDA REQUIRED)
```
I think `enable_language(Cuda)` already requires cuda anyway.